app/code/Magento/Checkout/Block/Checkout/AttributeMerger.php

```diff
@@ -269,9 +269,10 @@ protected function isFieldVisible($attributeCode, array $attributeConfig, array
      * @param array $attributeConfig
      * @param string $providerName name of the storage container used by UI component
      * @param string $dataScopePrefix
+     * @param array $additionalConfig
      * @return array
      */
-    protected function getMultilineFieldConfig($attributeCode, array $attributeConfig, $providerName, $dataScopePrefix)
+    protected function getMultilineFieldConfig($attributeCode, array $attributeConfig, $providerName, $dataScopePrefix, array $additionalConfig)
```

Comment:
Unfortunately, we are not allowed to change protected method signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
Ok, so could you please advise what would be correct approach in this case?

Reply:
@sidolov do you have any tip how this could be achieved?

Reply:
@berbua I will contact you :)